packages/flat-stores/package.json

```diff
@@ -14,7 +14,7 @@
     "white-web-sdk": ">=2.16"
   },
   "devDependencies": {
-    "@netless/fastboard": "1.0.0-canary.6",
+    "@netless/fastboard": "1.0.0-canary.7",
```

Comment:
Split to another PR

Reply:
This upgrade just updates some styles, so I guess it is safe to be included here.